meshtastic/interfaces/ble/lifecycle_service.py

```
@@ -0,0 +1,797 @@
+                        logger.debug(
+                            "Interface lost ownership during connect(), cleaning up gate claim for %s",
+                            getattr(connected_client, "address", "unknown"),
+                        )
+                    iface._connection_alias_key = None
+                    needs_cleanup = True
+            if needs_cleanup:
+                iface._mark_address_keys_disconnected(
+                    connected_device_key, connection_alias_key
+                )
+        else:
+            if is_closing:
```

Ruff (0.15.5)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)